src/ontology/edits/oeo-physical.omn

```diff
@@ -269,6 +269,56 @@ Class: <http://openenergy-platform.org/ontology/oeo-physical/ChemicalEnergy>
         Energy
     
     
+Class: <http://openenergy-platform.org/ontology/oeo-physical/FossilCombustionFuel>
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A FossilCombustionFuel is a CombustionFuel with the origin fossil.",
+        rdfs:comment "This class is used to implement the subset relationship between origins fossil and geogenic: Every object with the origin fossil also has the origin geogenic."
```

Comment:
If there is indeed a subset relationship between origins fossil and geogenic, isn't it perhaps better to represent the origins as classes (i.e. classified beneath the class 'origin') rather than as individuals?

Reply:
Yes we should just discuss that in a new issue because more than one issue add origins right now

Reply:
Okay, I will create an issue now

Reply:
Created #240